src/lib/network/clients/save_network.rb

```diff
@@ -77,12 +79,12 @@ def CopyConfiguredNetworkFiles
         file = receipt[:dir] + receipt[:file]
         adjust_for_network_disks(file) if file.include?("ifcfg-")
 
-        copy_from = String.Quote(file)
-        copy_to = String.Quote(inst_dir + receipt[:dir])
+        copy_from = file
+        copy_to = inst_dir + receipt[:dir]
 
         log.info("Copying #{copy_from} to #{copy_to}")
 
-        cmd = "cp " << copy_from << " " << copy_to
```

Comment:
BTW the old code was wrong, `String.Quote` needs the caller to provide apostrophes around its result:

Reply:
yes, I already noticed it. That is reason why ideally I would like to use Yast::Execute which prevents it by default, but there is other problems with it ( like exception or popup ) which does not appear with that SCR one. So I worry if I start replacing it in network, I can easily break it quite lot.